src/04-modality-specific-files/10-functional-near-infrared-spectroscopy.md

```diff
@@ -0,0 +1,367 @@
+may be used in `*_channels.tsv` to describe which channels were specified as short.
+
+Generic fields that MUST be present:
+
+| **Key name** | **Requirement level** | **Data type** | **Description**                                                                                                                                                                                                                                                                                                                                                                    |
+| ------------ | --------------------- | ------------- | ---------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| TaskName     | REQUIRED              | [string][]    | Name of the task. No two tasks should have the same name. The task label included in the file name is derived from this TaskName field by removing all non-alphanumeric (`[a-zA-Z0-9]`) characters. For example `TaskName` `"faces n-back"` will correspond to task label `facesnback`. A RECOMMENDED convention is to name resting state task using labels beginning with `rest`. |
+
+Generic fields which SHOULD be present: For consistency between studies and institutions, we encourage users to extract the values of these fields from the actual raw data. Whenever possible, please avoid using ad hoc wording.
+
+| **Key name**               | **Requirement level** | **Data type**                        | **Description**                                                                                                                                                                                                                                                                                                                        |
+| -------------------------- | --------------------- | ------------------------------------ | -------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| InstitutionName            | RECOMMENDED           | [string][]                           | The name of the institution in charge of the equipment that produced the composite instances.                                                                                                                                                                                                                                          |
```

Comment:
What's a "composite instance"? Same comment for the Description column in the next row.

Reply:
Apparently its an fMRI thing. Who knew. I should change it. And perhaps throughout the rest of BIDS.

https://www.medicalconnections.co.uk/kb/Composite-Instance